.github/workflows/update-label-backport-pr.yaml

```diff
@@ -22,12 +22,27 @@
         runs-on: ubuntu-latest
         permissions:
           pull-requests: write # Adding and removing labels
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          body: ${{ inputs.pr-body }}
         steps:
-          - name: Update labels in backported PRs
+          - name: Pre-process PR body
+            id: pre-process
+            uses: actions/github-script@v6.4.1
+            with:
+              script: |
+                const { body } = process.env
+                return body.replace(/\'/g, '')
+                  .replace(/"/g, '')
+                  .replace(/`/g, '')
+                  .replace(/$/g, '')
+              result-encoding: string
+
+          - name: Update labels
             env:
```

Comment:
Do we still need this section?

Reply:
I removed the `GITHUB_TOKEN` in the `env` section above and left it only here.